rs/sns/governance/api/src/ic_sns_governance.pb.v1.rs

```diff
@@ -613,6 +613,47 @@ pub struct ManageDappCanisterSettings {
     #[prost(uint64, optional, tag = "7")]
     pub wasm_memory_limit: ::core::option::Option<u64>,
 }
+#[derive(
+    candid::CandidType,
+    candid::Deserialize,
+    comparable::Comparable,
+    Clone,
+    PartialEq,
+    ::prost::Message,
+)]
+pub struct SnsVersion {
+    /// The hash of the Governance canister WASM.
+    #[prost(bytes = "vec", optional, tag = "1")]
+    pub governance_wasm_hash: ::core::option::Option<::prost::alloc::vec::Vec<u8>>,
+    /// The hash of the Swap canister WASM.
+    #[prost(bytes = "vec", optional, tag = "2")]
+    pub swap_wasm_hash: ::core::option::Option<::prost::alloc::vec::Vec<u8>>,
+    /// The hash of the Root canister WASM.
+    #[prost(bytes = "vec", optional, tag = "3")]
+    pub root_wasm_hash: ::core::option::Option<::prost::alloc::vec::Vec<u8>>,
+    /// The hash of the Index canister WASM.
+    #[prost(bytes = "vec", optional, tag = "4")]
+    pub index_wasm_hash: ::core::option::Option<::prost::alloc::vec::Vec<u8>>,
+    /// The hash of the Ledger canister WASM.
+    #[prost(bytes = "vec", optional, tag = "5")]
+    pub ledger_wasm_hash: ::core::option::Option<::prost::alloc::vec::Vec<u8>>,
+    /// The hash of the Ledger Archive canister WASM.
+    #[prost(bytes = "vec", optional, tag = "6")]
+    pub archive_wasm_hash: ::core::option::Option<::prost::alloc::vec::Vec<u8>>,
+}
```

Comment:
Isn't there already a `Version` struct? If `SnsVersion` is different, could there be a comment explaining the differences?

Reply:
It's the same for now, but it might become different in the future (e.g., we could have a shorthand notation such as `v1`, `v2` instead of having to specify the entire vec of hashes. 

In general, I think it makes more sense to justify why a non-primitive type is shared between APIs rather than why it's not shared.

Reply:
Added comments, as requested.